doc/source/components/files.txt

```diff
@@ -86,6 +86,12 @@ Creates a file. The main parameter for File is the target path. A ``File`` insta
 
     Source of symlink (for :py:attr:`ensure` = 'symlink')
 
+.. py:attribute:: sensitive_data
+
+    Mark a file as sensitive so its content is not exposed by the
+    (diff-)output of batou. This is useful in situation where the
```

Comment:
situations

Reply:
Just in case you did not know it ...

You can fix minor things directly here when you hit the `edit` button, when - and that's the default - the contributor checked "allow edits from maintainer".

Reply:
Yeah I've seen this before, but I'm not seeing any edit button currently, so ... :)

Reply:
It is hidden behind the three dots on the "Files changed" view - obviously :-/

![Screenshot from 2021-01-22 08-57-31](https://user-images.githubusercontent.com/9895620/105463150-eeb74900-5c8f-11eb-9ecc-e3c57700df89.png)